empyrical/tests/test_stats.py

```diff
@@ -837,3 +837,57 @@ def test_tail_ratio(self, returns, expected):
             empyrical.tail_ratio(returns),
             expected,
             1)
+
+    # Regression tests for CAGR.
+    @parameterized.expand([
```

Comment:
In addition, I think it would be good to have an obvious test case, e.g. have yearly returns of 30% 3 years in a row: `(.3*.3*.3)**(1/3) = 0.3`.